Bottom: d2f26889f7e54a87acdb5529bdcd4c6c1e8fb304
Top:    233cdaf60342b509b39fa3d0ea9c117062ec9e5d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2010-03-11 23:47:43 +0000

Allow running of action scripts when running PCB as an exporter


---

diff --git a/src/main.c b/src/main.c
index 162a0e0..e3c164f 100644
--- a/src/main.c
+++ b/src/main.c
@@ -996,12 +996,6 @@ main (int argc, char *argv[])
       LayerStringToLayerStack (Settings.InitialLayerStack);
     }
 
-  if (gui->printer || gui->exporter)
-    {
-      gui->do_export (0);
-      exit (0);
-    }
-
   /*    FIX_ME
      LoadBackgroundImage (Settings.BackgroundImage); */
 
@@ -1034,6 +1028,12 @@ main (int argc, char *argv[])
       hid_parse_actions (Settings.ActionString, 0);
     }
 
+  if (gui->printer || gui->exporter)
+    {
+      gui->do_export (0);
+      exit (0);
+    }
+
 #if HAVE_DBUS
   pcb_dbus_setup();
 #endif
